components/tikv_util/src/collections.rs

```diff
@@ -1,8 +1,9 @@
 // Copyright 2017 TiKV Project Authors. Licensed under Apache-2.0.
 
-pub type HashMap<K, V> = hashbrown::HashMap<K, V, hashbrown::hash_map::DefaultHashBuilder>;
-pub type HashSet<T> = hashbrown::HashSet<T, hashbrown::hash_map::DefaultHashBuilder>;
-pub use hashbrown::hash_map::Entry as HashMapEntry;
+use std::collections::hash_map::RandomState;
```

Comment:
No need for them any more.

Reply:
I am afraid that there will be extra work if we need to migrate to a better implementation (which is not imported to stdlib) in the future.

Reply:
I think if we still need `tikv_util::collections` then changing it to std or leaving to use hashbrown doesn't have differences. While if we switch to std totally then code can be simplified.

@BusyJay What do you think?

Reply:
Let's keep it in tikv_util. HashMap has been changed to several different implementations already, there may still be changes in the future.

Reply:
@breeswish Agree with you. If we still keep the abstraction, it's useless to change to use std HashMap.

Now, I am inclined to remove the abstraction and change to use std::collections::HashMap in tikv repo. This makes it more friendly to new contributors (they might always use the HashMap in std). And if a new implementation exposes the same interface as the std one, I think it should be a simple search and replace work.

Reply:
Note that [`hashbrown::hash_map::DefaultHashBuilder`](https://docs.rs/hashbrown/0.3.0/hashbrown/hash_map/type.DefaultHashBuilder.html) is the FxHasher while the standard HashMap is still using [SipHasher13](https://github.com/rust-lang/rust/blob/3991285f55a4b7cd92b7ffcdc396a3023076f5cb/src/libstd/collections/hash/map.rs#L2462). So you'll at least need a type alias anyway to keep using FxHasher.